Vision/Vision.bs

```diff
@@ -54,19 +41,25 @@ Boilerplate: omit conformance
 		to represent the emergent consensus of most of our participants.
 	* Be timeless enough that it does not need frequent revision.
 
+# The Mission of W3C # {#mission}
+
+	The W3C leads the community 
+	in defining a World Wide Web that puts users first, 
+	by developing protocols and guidelines 
+	to empower an ethical, equitable, informed, and interconnected society.
+
```

Comment:
Added mission statement.

Reply:
I think the addition of this section deserves its own issue and PR for discussion. I 100% agree with adding such a section. I think it's going in a good general direction. But I think we have a bit more to discuss on it before we declare it as our replacement for the existing mission statement, and I don't want to entangle getting consensus on this with the rest of the proposed improvements here.